src/server/utilities/onEnvironment/index.js

```diff
@@ -0,0 +1,9 @@
+const onEnvironment = (env, params) => wrappedFunction => {
+  const shouldCallFunction = process.env.SIMORGH_APP_ENV === env;
+
+  if (!shouldCallFunction) return;
+
+  wrappedFunction(params);
```

Comment:
Will there be any case where we would be expecting the return value of the function call ?

Reply:
Potentially, this function can be expanded upon at that time.